### DISCUSSION TITLE
Epiverse-TRACE developer space - Convert Your R Function to an S3 Generic: Benefits, Pitfalls & Design Considerations

### DISCUSSION THREAD

**Discussion by giscus** [epiverse-trace/epiverse-trace.github.io#16] [Blog post discussion]
# Epiverse-TRACE developer space - Convert Your R Function to an S3 Generic: Benefits, Pitfalls & Design Considerations

https://epiverse-trace.github.io/posts/s3-generic/

**Comment by Bisaloo** [comment]
It is interesting seeing this change happening in base R with a proposal to make `%*%` an S3 generic: https://bugs.r-project.org/show_bug.cgi?id=18483
